python/Ganga/GPIDev/Lib/Registry/PrepRegistry.py

```diff
@@ -185,13 +185,13 @@ def lookup(self, sharedir, unprepare=False):
         The optional parameter 'unprepare=True' can be set to call the unprepare method 
         on the returned objects.
         """
-        from Ganga.GPI import jobs, box, tasks
+        from Ganga.Core.GangaRepository import getRegistryProxy
         objectlist = []
-        for thing in jobs.select():
+        for thing in getRegistryProxy('jobs').select():
```

Comment:
See above, I don't want to fix/change the code behavior here I want to fix the bad GPI calls and change the behavior later.
Although in this case, yes an un-proxied repo is probably fine.